alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -9,6 +9,7 @@ crd_patches =$(subst $(CRD_SCHEMA_GEN_MANIFESTS),$(CRD_SCHEMA_GEN_OUTPUT),$(wild
 # $1 - crd file
 # $2 - patch file
 define patch-crd
+	cp -n $(CRD_SCHEMA_GEN_MANIFESTS)/$(notdir $2) '$(CRD_SCHEMA_GEN_OUTPUT)/' || true  # FIXME: centos
```

Comment:
what is `$(notdir $2)`?

Reply:
just the filename of the patch file, in the case that `crd_patches` passes the patch files' locations in a temp dir (ie, verify)